traefik/templates/deployment.yaml

```diff
@@ -88,6 +88,9 @@ spec:
           {{- if $config.hostPort }}
           hostPort: {{ $config.hostPort }}
           {{- end }}
+          {{- if $config.hostIP }}
+          hostIP: {{ $config.hostIP }}
```

Comment:
no exmaple in `values.yaml`?

Reply:
> To be honest, I'm not too sure whether this a use-case mainly considered or not

It might be a bit of an edge use case, but it's still an use case. Here it is:
- I have a server with a public-facing IP/network and a private-facing one
- I would like to have a private ingress controller and a public ingress controller

Without specifiying the `hostIP`, the first controller to start up "sits" on the port and makes it impossible to run the second controller without much magic and fiddling with the iptables manually.

> I don't see that value in the `values.yaml` tough, can you add it please.

Example added to `values.yaml`: https://github.com/containous/traefik-helm-chart/pull/229/files#diff-76ff6debe949af52b1679bce6b97cac0R115

> Also, please rebase
Sorry, onto which branch? My branch is exactly one commit ahead of `containous:master`. Thanks.